cypress/integration/specialFeatures/cookieBanner/index.js

```diff
@@ -47,17 +46,17 @@ Object.keys(config)
 
     const paths = getPaths(service);
     paths.forEach(path => {
-      describeForEuOnly(`${path} - Canonical Cookie Banner`, () => {
-        if (service !== 'news') {
+      if (service !== 'news') {
```

Comment:
Not quite sure why we are not running these canonical tests for News... is it because technically Simorgh does not render canonical news (and therefore there's no point testing it?)

Reply:
That sounds like a reasonable explanation why